_health-care/_js/_form.js

```diff
@@ -1,21 +1,47 @@
 var HealthApp = (function() {
   "use strict";
 
-  var identityTransform = function(formData, arg) { return arg; };
-  var extractTransform = function(formData, arg) { return formData[arg]; };
+  function identityTransform(formData, arg) { return arg; }
+  function extractTransform(formData, arg) { return formData[arg]; }
```

Comment:
function(foo) {} <-- creates an anonymous function that conceptually takes a closure of all the variables visible at that time. That closure uses memory itself and also keeps those variables alive. These little transformers don't actually reference anything outside their own scope so it probably actually gets optimized out, but I didn't want to rely on it.

Therefore I went back to bare named functions  (function iHaveAName(foo) {}) which does not create a closure.